### PR TITLE
docs(topology): Add drag and drop demo

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
**What**: Closes https://github.com/patternfly/patternfly-extension-issues/issues/17

### CODE REVIEW

**Comment by nicolethoen** [comment]
```
/home/runner/work/patternfly-react/patternfly-react/packages/react-topology/src/components/TopologyView/examples/TopologyDragDropDemo.md:96:34: 
  Cannot find module '../../../behavior' or its corresponding type declarations.
```

**Comment by jeff-phillips-18** [comment]
@nicolethoen Checks pass now

File: packages/react-topology/src/components/TopologyView/examples/TopologyPipelinesGettingStarted.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ import './topology-pipelines-example.css';
 ```ts file='./TopologyPipelinesGettingStartedDemo.tsx'
 ```
 
-#### getSpacerNodes
+## Functions
```
Comment (jenny-s51) [inline packages/react-topology/src/components/TopologyView/examples/TopologyPipelinesGettingStarted.md:70]: 👍

**Review by jenny-s51 — APPROVED** [review]
Docs and demo LGTM

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**